server/controllers/locations/getSuperUserRecords.js

```diff
@@ -0,0 +1,29 @@
+const boom = require("@hapi/boom");
+
+const connectToDatabase = require("../../database/dbConnection");
+const getSuperUserUsers = require("../../database/queries/user/getSuperUserUsers");
+const getRentalRecordsByIds = require("../../database/queries/rentalRecord/getRentalRecordsByIds");
+const isValidMongoID = require("../../utils/isValidMongoID");
+
+module.exports = async function getLocations(req, res, next) {
+  const { id: userId } = req.params;

```

Comment:
Use a stub function here to get a super user's ID. Once auth is implemented we should be able to get the current logged in user's ID on the backend